tap_indeedsponsoredjobs/streams.py

```diff
@@ -57,6 +57,15 @@ def get_child_context(self, record: dict, context: Optional[dict]) -> dict:
         return {
             "_sdc_employer_id": record["id"],
         }
+    
+    def get_records(self, context: Optional[dict]) -> Iterable[dict]:
+        """Return a generator of row-type dictionary objects."""
+        # If employer_ids is empty, null, or nonexistant, default to using the API to
+        # determine all employers.
+        if "employer_ids" in self.config and self.config["employer_ids"]:
```

Comment:
fyi `self.config.get("employer_ids")`  does what you want here and it's a pit more "pythonic"